### PR TITLE
Add translations No. 1, No. 2

### PR DESCRIPTION
**Pull request by Echolon**
Hello @anurodhp & @tmolitor-stud-tu,

finally my first commit of the translation macros.

If you agree to this. I will continue on the other file and add those strings.

Kind regards and looking forward!

rel. #144

### CODE REVIEW

**Comment by tmolitor-stud-tu** [comment]
Would make sense to only create a pull request if you are done with the work so that it doesn't get merged half way through.

Or at least mark it as "WIP" (work in progress)  in the title till it is finished completely.

**Comment by anurodhp** [comment]
Actually I would advocate doing it in small batches ( a screen it two)  and we can do multiple merges. This way it is not a large delta and less likely to have conflicts as other development is done

**Comment by Echolon** [comment]
@anurodhp @tmolitor-stud-tu 

Yes I agree with Anu. I cannot get things done im one day, or maybe even not in one week. And you can merge step-by-step and review the chnages. Also you can ensure I did messed up things.

So, I think I will create several pull request from now on. But to satsify Thilo I will put several files in per merge request

**Comment by Echolon** [comment]
@anurodhp @tmolitor-stud-tu ready for review

**Comment by Echolon** [comment]
I messed up with #405 - sorry

File: Monal/Classes/ActiveChatsViewController.m [hunk 1]
```diff
@@ -229,8 +229,8 @@ -(void) viewDidAppear:(BOOL)animated
...
-            UIAlertAction *acceptAction =[UIAlertAction actionWithTitle:@"Got it!" style:UIAlertActionStyleDefault handler:^(UIAlertAction *action) {
+            UIAlertController *messageAlert =[UIAlertController alertControllerWithTitle:NSLocalizedString(@"Notification Changes",@ "") message:[NSString stringWithFormat:NSLocalizedString(@"Notifications have changed in iOS 13 because of some iOS changes. For now you will just see something saying there is a new message and not the text or who sent it. I have decided to do this so you have reliable messaging while I work to update Monal to get the old expereince back.",@ "")] preferredStyle:UIAlertControllerStyleAlert];
+            UIAlertAction *acceptAction =[UIAlertAction actionWithTitle:NSLocalizedString(@"Got it!" style:UIAlertActionStyleDefault handler:^(UIAlertAction *action) {
```
Comment (tmolitor-stud-tu) [inline Monal/Classes/ActiveChatsViewController.m:233]: missing the `, @"")`

File: Monal/Classes/ContactsViewController.m [hunk 2]
```diff
@@ -122,15 +122,15 @@ -(void) showCallRequest:(NSNotification *) notification
         NSString *userName=[dic objectForKey:kUsername];
         
         
-        UIAlertController *messageAlert =[UIAlertController alertControllerWithTitle:@"Incoming Call" message:[NSString stringWithFormat:@"Incoming audio call to %@ from %@ ",userName,  contactName] preferredStyle:UIAlertControllerStyleAlert];
-        UIAlertAction *acceptAction =[UIAlertAction actionWithTitle:@"Accept" style:UIAlertActionStyleDefault handler:^(UIAlertAction *action) {
+        UIAlertController *messageAlert =[UIAlertController alertControllerWithTitle:NSLocalizedString(@"Incoming Call" message:[NSString stringWithFormat:NSLocalizedString(@"Incoming audio call to %@ from %@ ",@""),userName,  contactName] preferredStyle:UIAlertControllerStyleAlert];
```
Comment (tmolitor-stud-tu) [inline Monal/Classes/ContactsViewController.m:125]: missing `, @"")` again

File: Monal/Classes/MLBaseCell.m [hunk 3]
```diff
@@ -7,8 +7,8 @@
 //
 
 #import "MLBaseCell.h"
-NSString *const kDelivered=@"Delivered";
-NSString *const kRead=@"Read";
+NSString *const kDelivered=NSLocalizedString(@"Delivered",@"");
```
Comment (tmolitor-stud-tu) [inline Monal/Classes/MLBaseCell.m:10]: these are keys, don't translate them

File: Monal/Classes/MLComposeViewController.m [hunk 4]
```diff
@@ -28,8 +28,8 @@ - (IBAction)send:(id)sender
 {
     if([[MLXMPPManager sharedInstance].connectedXMPP count]==0)
     {
-        UIAlertController *messageAlert =[UIAlertController alertControllerWithTitle:@"No connected accounts" message:@"Please make sure at least one account has connected before trying to message someone." preferredStyle:UIAlertControllerStyleAlert];
-        UIAlertAction *closeAction =[UIAlertAction actionWithTitle:@"Close" style:UIAlertActionStyleCancel handler:^(UIAlertAction *action) {
+        UIAlertController *messageAlert =[UIAlertController alertControllerWithTitle:NSLocalizedString(@"No connected accounts",@"") message:NSLocalizedString(@"Please make sure at least one account has connected before trying to message someone.",@"") preferredStyle:UIAlertControllerStyleAlert];
+        UIAlertAction *closeAction =[UIAlertAction actionWithTitle:@"Close",@"") style:UIAlertActionStyleCancel handler:^(UIAlertAction *action) {
```
Comment (tmolitor-stud-tu) [inline Monal/Classes/MLComposeViewController.m:32]: `NSLocalizedString(` missing

File: Monal/Classes/MLComposeViewController.m [hunk 5]
```diff
@@ -40,8 +40,8 @@ - (IBAction)send:(id)sender
         
         if(self.message.text.length==0)
         {
-            UIAlertController *messageAlert =[UIAlertController alertControllerWithTitle:@"Error" message:@"Message can't be empty" preferredStyle:UIAlertControllerStyleAlert];
-            UIAlertAction *closeAction =[UIAlertAction actionWithTitle:@"Close" style:UIAlertActionStyleCancel handler:^(UIAlertAction *action) {
+            UIAlertController *messageAlert =[UIAlertController alertControllerWithTitle:NSLocalizedString(@"Error",@"") message:NSLocalizedString(NSLocalizedString(@"Message can't be empty",@"") preferredStyle:UIAlertControllerStyleAlert];
+            UIAlertAction *closeAction =[UIAlertAction actionWithTitle:@"Close",@"") style:UIAlertActionStyleCancel handler:^(UIAlertAction *action) {
```
Comment (tmolitor-stud-tu) [inline Monal/Classes/MLComposeViewController.m:44]: `NSLocalizedString(` missing here, too

File: Monal/Classes/MLImageManager.m [hunk 6]
```diff
@@ -167,11 +167,11 @@ -(void) setIconForContact:(NSString*) contact andAccount:(NSString*) accountNo W
 
     if([[EncodingTools dataWithBase64EncodedString:data] writeToFile:writablePath atomically:NO] )
     {
-        DDLogVerbose(@"wrote imgae to file");
+        DDLogVerbose(@"wrote image to file",@""));
```
Comment (tmolitor-stud-tu) [inline Monal/Classes/MLImageManager.m:170]: `,@"")` should be removed

File: Monal/Classes/MLImageManager.m [hunk 7]
```diff
@@ -167,11 +167,11 @@ -(void) setIconForContact:(NSString*) contact andAccount:(NSString*) accountNo W
...
     }
     else
     {
-        DDLogError(@"failed to write image");
+        DDLogError(@"failed to write image",@""));
```
Comment (tmolitor-stud-tu) [inline Monal/Classes/MLImageManager.m:174]: `,@"")` should be removed

File: Monal/Classes/MLLogInViewController.m [hunk 8]
```diff
@@ -76,8 +76,8 @@ -(IBAction) login:(id)sender
     if(!user || !domain)
     {
         self.loginHUD.hidden=YES;
-        UIAlertController *alert = [UIAlertController alertControllerWithTitle:@"Invalid Credentails" message:@"Your XMPP account should be in in the format user@domain. For special configurations, use manual setup." preferredStyle:UIAlertControllerStyleAlert];
-        [alert addAction:[UIAlertAction actionWithTitle:@"Close" style:UIAlertActionStyleDefault handler:^(UIAlertAction * _Nonnull action) {
+        UIAlertController *alert = [UIAlertController alertControllerWithTitle:NSLocalizedString(@"Invalid Credentails",@"") message:NSLocalizedString(@"Your XMPP account should be in in the format user@domain. For special configurations, use manual setup.",@"") preferredStyle:UIAlertControllerStyleAlert];
+        [alert addAction:[UIAlertAction actionWithTitle:NSLocalizedString(@"Close" style:UIAlertActionStyleDefault handler:^(UIAlertAction * _Nonnull action) {
```
Comment (tmolitor-stud-tu) [inline Monal/Classes/MLLogInViewController.m:80]: `, @"")` missing

File: Monal/Classes/chatViewController.m [hunk 9]
```diff
@@ -839,16 +839,16 @@ -(void) presentMucInvite:(NSNotification *)notification
...
-    NSString *from = [userDic objectForKey:@"from"];
+    NSString *from = [userDic objectForKey:=NSLocalizedString(@"from",@ "")];
     dispatch_async(dispatch_get_main_queue(), ^{
-        NSString* messageString = [NSString  stringWithFormat:NSLocalizedString(@"You have been invited to a conversation %@?", nil), from ];
-        UIAlertController *alert = [UIAlertController alertControllerWithTitle:@"Group Chat Invite" message:messageString preferredStyle:UIAlertControllerStyleAlert];
+        NSString* messageString = [NSString  stringWithFormat:NSLocalizedString(=NSLocalizedString(@"You have been invited to a conversation %@?",@ ""), nil), from ];
+        UIAlertController *alert = [UIAlertController alertControllerWithTitle:=NSLocalizedString(@"Group Chat Invite" message:messageString preferredStyle:UIAlertControllerStyleAlert];
         
-        [alert addAction:[UIAlertAction actionWithTitle:@"Join" style:UIAlertActionStyleDefault handler:^(UIAlertAction * _Nonnull action) {
+        [alert addAction:[UIAlertAction actionWithTitle:=NSLocalizedString(@"Join" style:UIAlertActionStyleDefault handler:^(UIAlertAction * _Nonnull action) {
```
Comment (tmolitor-stud-tu) [inline Monal/Classes/chatViewController.m:847]: missing `, @"")`

File: Monal/Classes/chatViewController.m [hunk 10]
```diff
@@ -839,16 +839,16 @@ -(void) presentMucInvite:(NSNotification *)notification
...
             [xmppAccount joinRoom:from withNick:xmppAccount.connectionProperties.identity.user andPassword:nil];
             [alert dismissViewControllerAnimated:YES completion:nil];
         }]];
-        [alert addAction:[UIAlertAction actionWithTitle:@"Close" style:UIAlertActionStyleCancel handler:^(UIAlertAction * _Nonnull action) {
+        [alert addAction:[UIAlertAction actionWithTitle:=NSLocalizedString(@"Close" style:UIAlertActionStyleCancel handler:^(UIAlertAction * _Nonnull action) {
```
Comment (tmolitor-stud-tu) [inline Monal/Classes/chatViewController.m:851]: missing `, @"")`

**Comment by anurodhp** [comment]
Well have to resolve the conflict  here and in the other pr

**Comment by Echolon** [comment]
@anurodhp can you have a look? maybe not too long waiting if possible, I had to rebase already

**Pull request merged**